### DISCUSSION TITLE
[Feature] Default quota

### DISCUSSION THREAD

**Discussion by DenuxPlays** [immich-app/immich#7300] [Feature Request]
### The feature

Hey,
I've added immich to my SSO system but now I have the problem that a newly registered user has an unlimited (or limited by the servers storage).
Is there a way to define a default storage?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by samholton** [comment]
Going to take a stab at this one under https://github.com/immich-app/immich/discussions/7399

**Reply by samholton** [reply]
https://github.com/immich-app/immich/pull/7548
